src/backend.ts

```diff
@@ -157,13 +157,7 @@ export class Backend<M extends IAnyModelType> {
       console.error("Unexpected error during process:", e);
       return;
     }
-    const completeProcessResult: ProcessResult = {
-      updates: [],
-      accessUpdates: [],
-      errorValidations: [],
-      warningValidations: [],
-      ...processResult
-    };
+    const completeProcessResult: ProcessResult = processResult;
```

Comment:
With the new typescript version this line generated an error. The `ProcessResult` already has all the values present, so typescript was complaining the given keys where always going to be overridden.